tests/php/core-api/wpcom-fields/test_class.subscribers-endpoint.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+require_once JETPACK__PLUGIN_DIR . '/tests/php/lib/class-wp-test-jetpack-rest-testcase.php';
+require_once JETPACK__PLUGIN_DIR . '/tests/php/lib/class-wp-test-spy-rest-server.php';
```

Comment:
What does this library do? 
Do we need it here?

Reply:
Good catch. I had initially copied the publicize test file. I'll remove it.